Source/ThirdParty/jsep.js

```diff
@@ -582,7 +582,7 @@ define(function() {
 	/**
 	 * @method jsep.addLiteral
```

Comment:
Don't update anything in the `ThirdParty` directory- It's third party code, and we don't want to make changes unless necessary.